### PR TITLE
Remove alculquicondor from job approvers

### PR DESCRIPTION
**Pull request by alculquicondor**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

I will be reducing my involvement in the kubernetes project.

Thank you @soltysh for the opportunity to contribute to the job controller.

And thanks @atiratree and @mimowo for stepping up in contributions.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/controller/job/OWNERS [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 approvers:
   - sig-apps-approvers
-  - alculquicondor
```
Comment (aojea) [inline pkg/controller/job/OWNERS:5]: we have emeritus , you deserve it

Comment (aojea) [reply pkg/controller/job/OWNERS:5]:
```
emeritus_approvers:
  - alculquicondor
 ```

Comment (alculquicondor) [reply pkg/controller/job/OWNERS:5]: I forgot about that, thanks :)

Just here is enough, we don't need to repeat in the test folder.

**Comment by aojea** [comment]
/lgtm

👋

**Pull request merged**